src/components/NoteCard/NoteCard.js

```diff
@@ -1,8 +1,10 @@
 import "./NoteCard.css";
 
-const NoteCard = ({ title, body }) => {
+const NoteCard = ({ note, editNoteHandler }) => {
+  const { title, body, createdAt } = note;
```

Comment:
nice that you separated the destructuring of this model

Reply:
Thank you!